src/makernote_int.cpp

```diff
@@ -81,13 +84,15 @@ std::string getExiv2ConfigPath() {
   currentPath = std::string(pw ? pw->pw_dir : "");
 #endif
   return (currentPath / inifile).string();
+#endif
+  return "";
 }
 
 std::string readExiv2Config([[maybe_unused]] const std::string& section, [[maybe_unused]] const std::string& value,
                             const std::string& def) {
   std::string result = def;
 
-#ifdef EXV_ENABLE_INIH
+#if defined(EXV_ENABLE_INIH) and defined(EXV_ENABLE_FILESYSTEM)
```

Comment:
```suggestion
#if defined(EXV_ENABLE_INIH) && defined(EXV_ENABLE_FILESYSTEM)
```

Reply:
thanks, let's see if I finally get a 100% green this time :)